app/models/job_invocation.rb

```diff
@@ -139,6 +139,10 @@ def to_label
     description
   end
 
+  def latest_jobs(limit = 5)
+    JobInvocation.order(id: :desc).limit(limit)
```

Comment:
This requires permission check, there's `.authorized` named scope that can be used, just pass a permission name to it, the app/* contains many examples of use :-)